config.schema.yaml

```diff
@@ -0,0 +1,20 @@
+---
+appliance:
+  type: object
+  properties:
+    default:
```

Comment:
Thank you for your advice @LindsayHill . I corrected it in [this commit](https://github.com/StackStorm-Exchange/stackstorm-acos/pull/6/commits/bbe76c98e556573ad3a2f01f9ab09e76c88685c7). But I'm not sure that is the best one.

I understood that I should have to write the config.schema to be able to specify multiple configuration-sets (`target IP address`, `username` and `password`) to connect. And I corrected `config.schema.yaml` following the one of MySQL pack.

But I have no confidence that the corrected `config.schema.yaml` is right.

In the `st2 config mysql` command, the properties which are specified in the `config.schema.file` (e.g. `host`, `user`, ...) aren't asked interactively. This is counterintuitive, I guess.

I'm not familiar with the implementation to parse the configuration schema file. So I'll make sure to know it.
If you know the best way to do it, could you please tell me that?